src/libs/antares/series/include/antares/series/series.h

```diff
@@ -35,10 +35,17 @@ namespace Antares::Data
 class TimeSeries
 {
 public:
-    using numbers = Matrix<uint32_t>;
+    struct Numbers : public Matrix<uint32_t> // TODO[FOM] remove inheritance, it was added mostly to avoid breaking existing code
```

Comment:
Indeed, would be better to use composition here with a class encapsulating both TS numbers and registered timeseries.
Do you prefer to postpone it to later work?

Reply:
Yes I'd rather do it later

Reply:
@sylvlecl done [here](#2085)